### ISSUE TITLE
Traceback when setting breakpoint on lambda definition (CodeLLDB/Linux)

### ISSUE THREAD

**Issue by Aster89** [puremourning/vimspector#615]
### Description

I think it's #561, or a variation of it.

A screencast is better than a thousand words:
[![asciicast](https://asciinema.org/a/iFYwccurXpTMiVEst9SuTPsWv.svg)](https://asciinema.org/a/iFYwccurXpTMiVEst9SuTPsWv)

### Works in vim?

Reproducable in Vim

### Works with clean config?

Yes

### Minimal Reproduction

List of steps to reproduce:

1. Run `vim ---clean Nu /path/to/vimspector/support/minimal_vimrc`
2. Open a file with this content
```cpp
#include <iostream>
#include <functional>
#include <vector>

int main() {
    std::function<void()> actions = []{
        std::cout << "hello" << std::endl;
    };
}
```
3. Try adding a breakpoint on the line with `=`. Vimspector will actually insert the breakpoint on the line below, where there's `hello`.
4. Now open the breakpoints window and try toggling it. You should see errors.
5. Try toggling a few more times, and you should see something weird (the breakpoint gets listed twice) and errors. (This is not in the screencast.)
6. Now try doing anything else, like restarting, resetting, ... You should see errors.

### Expected Behaviour

No errors?

### Actual Behaviour

See repro steps.

### Installation Type

Plugin Manager (git clone)

### Vimspector version

b33b1be7679efb756c88ad398f6e7d77505c02c6

### Debug Info

```Text
Vimspector Debug Info
--------------------------------------------------------------------------------
ConnectionType: channel
Adapter: 
--------------------------------------------------------------------------------
{
  "command": [
    "/home/enrico/.vim/plugged/vimspector/gadgets/linux/CodeLLDB/adapter/codelldb",
    "--port",
    "48383"
  ],
  "configuration": {
    "args": [],
    "cargo": {},
    "cwd": "/home/enrico",
    "env": {},
    "name": "lldb",
    "terminal": "integrated",
    "type": "lldb"
  },
  "name": "CodeLLDB",
  "port": "48383",
  "type": "CodeLLDB",
  "env": {},
  "cwd": "/home/enrico"
}
--------------------------------------------------------------------------------
Configuration: 
--------------------------------------------------------------------------------
{
  "adapter": "CodeLLDB",
  "breakpoints": {
    "exception": {
      "cpp_throw": "N",
      "cpp_catch": "N"
    }
  },
  "configuration": {
    "request": "launch",
    "program": "uffa",
    "cwd": "/home/enrico",
    "externalConsole": true,
    "MIMode": "gdb",
    "setupCommands": [
      {
        "description": "Enable pretty-printing for gdb",
        "text": "-enable-pretty-printing",
        "ignoreFailures": true
      },
      {
        "description": "Auto load symbols when loading an .so file",
        "text": "set auto-solib-add",
        "ignoreFailures": false
      }
    ]
  }
}
--------------------------------------------------------------------------------
API Prefix: 
Launch/Init: True / True
Workspace Root: /home/enrico
Launch Config: 
--------------------------------------------------------------------------------
{
  "args": [],
  "cargo": {},
  "cwd": "/home/enrico",
  "env": {},
  "name": "lldb",
  "terminal": "integrated",
  "type": "lldb",
  "request": "launch",
  "program": "uffa",
  "externalConsole": true,
  "MIMode": "gdb",
  "setupCommands": [
    {
      "description": "Enable pretty-printing for gdb",
      "text": "-enable-pretty-printing",
      "ignoreFailures": true
    },
    {
      "description": "Auto load symbols when loading an .so file",
      "text": "set auto-solib-add",
      "ignoreFailures": false
    }
  ]
}
--------------------------------------------------------------------------------
Server Capabilities: 
--------------------------------------------------------------------------------
{
  "exceptionBreakpointFilters": [
    {
      "default": true,
      "filter": "cpp_throw",
      "label": "C++: on throw"
    },
    {
      "default": false,
      "filter": "cpp_catch",
      "label": "C++: on catch"
    }
  ],
  "supportTerminateDebuggee": true,
  "supportsCancelRequest": true,
  "supportsCompletionsRequest": true,
  "supportsConditionalBreakpoints": true,
  "supportsConfigurationDoneRequest": true,
  "supportsDataBreakpoints": true,
  "supportsDelayedStackTraceLoading": true,
  "supportsEvaluateForHovers": true,
  "supportsFunctionBreakpoints": true,
  "supportsGotoTargetsRequest": true,
  "supportsHitConditionalBreakpoints": true,
  "supportsLogPoints": true,
  "supportsReadMemoryRequest": true,
  "supportsRestartFrame": true,
  "supportsSetVariable": true
}
--------------------------------------------------------------------------------
Line Breakpoints: 
--------------------------------------------------------------------------------
{
  "/home/enrico/uffa.cpp": [
    {
      "state": "ENABLED",
      "line": 6,
      "options": {},
      "sign_id": 1
    },
    {
      "state": "ENABLED",
      "line": 7,
      "options": null
    }
  ]
}
--------------------------------------------------------------------------------
Func Breakpoints: 
--------------------------------------------------------------------------------
[]
--------------------------------------------------------------------------------
Ex Breakpoints: 
--------------------------------------------------------------------------------
{
  "filters": []
}
--------------------------------------------------------------------------------
```

### Log file

```Text
2022-07-02 20:44:35,249 - INFO - **** INITIALISING NEW VIMSPECTOR SESSION ****
2022-07-02 20:44:35,249 - INFO - API is: 
2022-07-02 20:44:35,249 - INFO - VIMSPECTOR_HOME = /home/enrico/.vim/plugged/vimspector
2022-07-02 20:44:35,250 - INFO - gadgetDir = /home/enrico/.vim/plugged/vimspector/gadgets/linux
2022-07-02 20:44:37,223 - INFO - User requested start debug session with {}
2022-07-02 20:44:37,225 - DEBUG - Reading configurations from: None
2022-07-02 20:44:37,242 - DEBUG - Reading configurations from: /home/enrico/.vimspector.json
2022-07-02 20:44:37,266 - DEBUG - Reading gadget config: /home/enrico/.vim/plugged/vimspector/gadgets/linux/.gadgets.json
2022-07-02 20:44:37,294 - DEBUG - Reading gadget config: None
2022-07-02 20:44:39,023 - INFO - Configuration: {"adapter": "CodeLLDB", "breakpoints": {"exception": {"cpp_throw": "N", "cpp_catch": "N"}}, "configuration": {"request": "launch", "program": "uffa", "cwd": "/home/enrico", "externalConsole": true, "MIMode": "gdb", "setupCommands": [{"description": "Enable pretty-printing for gdb", "text": "-enable-pretty-printing", "ignoreFailures": true}, {"description": "Auto load symbols when loading an .so file", "text": "set auto-solib-add", "ignoreFailures": false}]}}
2022-07-02 20:44:39,023 - INFO - Adapter: {"command": ["/home/enrico/.vim/plugged/vimspector/gadgets/linux/CodeLLDB/adapter/codelldb", "--port", "48383"], "configuration": {"args": [], "cargo": {}, "cwd": "/home/enrico", "env": {}, "name": "lldb", "terminal": "integrated", "type": "lldb"}, "name": "CodeLLDB", "port": "48383", "type": "CodeLLDB"}
2022-07-02 20:44:39,024 - DEBUG - min_width/height: 149/50, actual: 145/47 - result: horizontal
2022-07-02 20:44:39,047 - DEBUG - LAUNCH!
2022-07-02 20:44:39,048 - INFO - Starting debug adapter with: {"command": ["/home/enrico/.vim/plugged/vimspector/gadgets/linux/CodeLLDB/adapter/codelldb", "--port", "48383"], "configuration": {"args": [], "cargo": {}, "cwd": "/home/enrico", "env": {}, "name": "lldb", "terminal": "integrated", "type": "lldb"}, "name": "CodeLLDB", "port": "48383", "type": "CodeLLDB"}
2022-07-02 20:44:39,049 - DEBUG - Connection Type: channel
2022-07-02 20:44:39,102 - INFO - Debug Adapter Started
2022-07-02 20:44:39,103 - DEBUG - Sending Message: {"command": "initialize", "arguments": {"adapterID": "CodeLLDB", "clientID": "vimspector", "clientName": "vimspector", "linesStartAt1": true, "columnsStartAt1": true, "locale": "en_GB", "pathFormat": "path", "supportsVariableType": true, "supportsVariablePaging": false, "supportsRunInTerminalRequest": true, "supportsMemoryReferences": true}, "seq": 0, "type": "request"}
2022-07-02 20:44:39,366 - DEBUG - Message received: {'type': 'response', 'request_seq': 0, 'success': True, 'command': 'initialize', 'body': {'exceptionBreakpointFilters': [{'default': True, 'filter': 'cpp_throw', 'label': 'C++: on throw'}, {'default': False, 'filter': 'cpp_catch', 'label': 'C++: on catch'}], 'supportTerminateDebuggee': True, 'supportsCancelRequest': True, 'supportsCompletionsRequest': True, 'supportsConditionalBreakpoints': True, 'supportsConfigurationDoneRequest': True, 'supportsDataBreakpoints': True, 'supportsDelayedStackTraceLoading': True, 'supportsEvaluateForHovers': True, 'supportsFunctionBreakpoints': True, 'supportsGotoTargetsRequest': True, 'supportsHitConditionalBreakpoints': True, 'supportsLogPoints': True, 'supportsReadMemoryRequest': True, 'supportsRestartFrame': True, 'supportsSetVariable': True}}
2022-07-02 20:44:39,367 - DEBUG - Sending Message: {"command": "launch", "arguments": {"args": [], "cargo": {}, "cwd": "/home/enrico", "env": {}, "name": "lldb", "terminal": "integrated", "type": "lldb", "request": "launch", "program": "uffa", "externalConsole": true, "MIMode": "gdb", "setupCommands": [{"description": "Enable pretty-printing for gdb", "text": "-enable-pretty-printing", "ignoreFailures": true}, {"description": "Auto load symbols when loading an .so file", "text": "set auto-solib-add", "ignoreFailures": false}]}, "seq": 1, "type": "request"}
2022-07-02 20:44:39,368 - DEBUG - Message received: {'type': 'event', 'seq': 1, 'event': 'initialized'}
2022-07-02 20:44:39,368 - DEBUG - Sending Message: {"command": "setBreakpoints", "arguments": {"source": {"name": "uffa.cpp", "path": "/home/enrico/uffa.cpp"}, "breakpoints": [{"line": 6}], "sourceModified": false}, "seq": 2, "type": "request"}
2022-07-02 20:44:39,368 - DEBUG - Sending Message: {"command": "setFunctionBreakpoints", "arguments": {"breakpoints": []}, "seq": 3, "type": "request"}
2022-07-02 20:44:39,368 - DEBUG - Sending Message: {"command": "setExceptionBreakpoints", "arguments": {"filters": []}, "seq": 4, "type": "request"}
2022-07-02 20:44:39,369 - DEBUG - Message received: {'type': 'event', 'seq': 2, 'event': 'module', 'body': {'module': {'addressRange': 'FFFFFFFFFFFFFFFF', 'id': 'FFFFFFFFFFFFFFFF', 'name': 'uffa', 'path': '/home/enrico/uffa', 'symbolFilePath': '/home/enrico/uffa', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,369 - DEBUG - Message received: {'type': 'request', 'seq': 3, 'command': 'runInTerminal', 'arguments': {'args': ['/home/enrico/.vim/plugged/vimspector/gadgets/linux/download/CodeLLDB/v1.6.10/root/extension/adapter/codelldb', 'terminal-agent', '--port=45813'], 'cwd': '', 'kind': 'integrated', 'title': 'lldb'}}
2022-07-02 20:44:39,369 - DEBUG - Defaulting working directory to /home/enrico
2022-07-02 20:44:39,370 - DEBUG - Sending Message: {"seq": 5, "type": "response", "request_seq": 3, "command": "runInTerminal", "body": {"processId": 1396994}, "success": true}
2022-07-02 20:44:39,371 - DEBUG - Message received: {'type': 'response', 'request_seq': 2, 'success': True, 'command': 'setBreakpoints', 'body': {'breakpoints': [{'id': 1, 'line': 7, 'message': 'Resolved locations: 0', 'verified': True}]}}
2022-07-02 20:44:39,371 - DEBUG - Message received: {'type': 'response', 'request_seq': 3, 'success': True, 'command': 'setFunctionBreakpoints', 'body': {'breakpoints': []}}
2022-07-02 20:44:39,371 - DEBUG - Message received: {'type': 'response', 'request_seq': 4, 'success': True, 'command': 'setExceptionBreakpoints'}
2022-07-02 20:44:39,371 - DEBUG - Sending Message: {"command": "configurationDone", "seq": 6, "type": "request"}
2022-07-02 20:44:39,371 - DEBUG - Message received: {'type': 'response', 'request_seq': 6, 'success': True, 'command': 'configurationDone'}
2022-07-02 20:44:39,372 - DEBUG - Message received: {'type': 'event', 'seq': 4, 'event': 'output', 'body': {'output': 'Launching: /home/enrico/uffa\n'}}
2022-07-02 20:44:39,406 - DEBUG - Message received: {'type': 'event', 'seq': 5, 'event': 'output', 'body': {'output': 'Launched process 1397004\n'}}
2022-07-02 20:44:39,407 - DEBUG - Message received: {'type': 'response', 'request_seq': 1, 'success': True, 'command': 'launch'}
2022-07-02 20:44:39,407 - DEBUG - Sending Message: {"command": "threads", "seq": 7, "type": "request"}
2022-07-02 20:44:39,407 - DEBUG - Message received: {'type': 'event', 'seq': 6, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7FC6000', 'id': '7FFFF7FC6000', 'name': 'ld-linux-x86-64.so.2', 'path': '/usr/lib/ld-linux-x86-64.so.2', 'symbolFilePath': '/usr/lib/ld-linux-x86-64.so.2', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,407 - DEBUG - Message received: {'type': 'event', 'seq': 7, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7FC4000', 'id': '7FFFF7FC4000', 'name': '[vdso]', 'path': '[vdso]', 'symbolStatus': 'Symbols not found'}, 'reason': 'new'}}
2022-07-02 20:44:39,407 - DEBUG - Message received: {'type': 'event', 'seq': 8, 'event': 'breakpoint', 'body': {'breakpoint': {'id': 1, 'line': 7, 'message': 'Resolved locations: 1', 'verified': True}, 'reason': 'changed'}}
2022-07-02 20:44:39,407 - DEBUG - Message received: {'type': 'event', 'seq': 9, 'event': 'module', 'body': {'module': {'addressRange': '555555554000', 'id': '555555554000', 'name': 'uffa', 'path': '/home/enrico/uffa', 'symbolFilePath': '/home/enrico/uffa', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,407 - DEBUG - Message received: {'type': 'event', 'seq': 10, 'event': 'continued', 'body': {'allThreadsContinued': True, 'threadId': 0}}
2022-07-02 20:44:39,408 - DEBUG - Message received: {'type': 'response', 'request_seq': 7, 'success': True, 'command': 'threads', 'body': {'threads': [{'id': 1397004, 'name': '1: tid=1397004 "uffa"'}]}}
2022-07-02 20:44:39,408 - DEBUG - Message received: {'type': 'event', 'seq': 11, 'event': 'module', 'body': {'module': {'addressRange': 'FFFFFFFFFFFFFFFF', 'id': 'FFFFFFFFFFFFFFFF', 'name': 'ld-linux-x86-64.so.2', 'path': '/lib64/ld-linux-x86-64.so.2', 'symbolFilePath': '/lib64/ld-linux-x86-64.so.2', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,408 - DEBUG - Message received: {'type': 'event', 'seq': 12, 'event': 'module', 'body': {'module': {'id': 'FFFFFFFFFFFFFFFF', 'name': ''}, 'reason': 'removed'}}
2022-07-02 20:44:39,408 - DEBUG - Message received: {'type': 'event', 'seq': 13, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7FC4000', 'id': '7FFFF7FC4000', 'name': '[vdso]', 'path': '[vdso]', 'symbolStatus': 'Symbols not found'}, 'reason': 'new'}}
2022-07-02 20:44:39,489 - DEBUG - Message received: {'type': 'event', 'seq': 14, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7C00000', 'id': '7FFFF7C00000', 'name': 'libstdc++.so.6', 'path': '/usr/lib/libstdc++.so.6', 'symbolFilePath': '/usr/lib/libstdc++.so.6', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,490 - DEBUG - Message received: {'type': 'event', 'seq': 15, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7EB4000', 'id': '7FFFF7EB4000', 'name': 'libm.so.6', 'path': '/usr/lib/libm.so.6', 'symbolFilePath': '/usr/lib/libm.so.6', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,493 - DEBUG - Message received: {'type': 'event', 'seq': 16, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7E94000', 'id': '7FFFF7E94000', 'name': 'libgcc_s.so.1', 'path': '/usr/lib/libgcc_s.so.1', 'symbolFilePath': '/usr/lib/libgcc_s.so.1', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,497 - DEBUG - Message received: {'type': 'event', 'seq': 17, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7800000', 'id': '7FFFF7800000', 'name': 'libc.so.6', 'path': '/usr/lib/libc.so.6', 'symbolFilePath': '/usr/lib/libc.so.6', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,497 - DEBUG - Message received: {'type': 'event', 'seq': 18, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7C00000', 'id': '7FFFF7C00000', 'name': 'libstdc++.so.6', 'path': '/usr/lib/libstdc++.so.6', 'symbolFilePath': '/usr/lib/libstdc++.so.6', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,497 - DEBUG - Message received: {'type': 'event', 'seq': 19, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7EB4000', 'id': '7FFFF7EB4000', 'name': 'libm.so.6', 'path': '/usr/lib/libm.so.6', 'symbolFilePath': '/usr/lib/libm.so.6', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,497 - DEBUG - Message received: {'type': 'event', 'seq': 20, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7E94000', 'id': '7FFFF7E94000', 'name': 'libgcc_s.so.1', 'path': '/usr/lib/libgcc_s.so.1', 'symbolFilePath': '/usr/lib/libgcc_s.so.1', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,498 - DEBUG - Message received: {'type': 'event', 'seq': 21, 'event': 'module', 'body': {'module': {'addressRange': '7FFFF7800000', 'id': '7FFFF7800000', 'name': 'libc.so.6', 'path': '/usr/lib/libc.so.6', 'symbolFilePath': '/usr/lib/libc.so.6', 'symbolStatus': 'Symbols loaded.'}, 'reason': 'new'}}
2022-07-02 20:44:39,498 - DEBUG - Message received: {'type': 'event', 'seq': 22, 'event': 'output', 'body': {'output': 'Process exited with code 0.\n'}}
2022-07-02 20:44:39,498 - DEBUG - Message received: {'type': 'event', 'seq': 23, 'event': 'exited', 'body': {'exitCode': 0}}
2022-07-02 20:44:39,499 - INFO - User Msg: The debuggee exited with status code: 0
2022-07-02 20:44:39,500 - DEBUG - Message received: {'type': 'event', 'seq': 24, 'event': 'terminated', 'body': {}}
```

### Vim version

```Text
VIM - Vi IMproved 9.0 (2022 Jun 28, compiled Jun 28 2022 16:22:51)
Compiled by Arch Linux
Huge version with GTK3 GUI.  Features included (+) or not (-):
+acl               +file_in_path      +mouse_urxvt       -tag_any_white
+arabic            +find_in_path      +mouse_xterm       +tcl/dyn
+autocmd           +float             +multi_byte        +termguicolors
+autochdir         +folding           +multi_lang        +terminal
-autoservername    -footer            -mzscheme          +terminfo
+balloon_eval      +fork()            +netbeans_intg     +termresponse
+balloon_eval_term +gettext           +num64             +textobjects
+browse            -hangul_input      +packages          +textprop
++builtin_terms    +iconv             +path_extra        +timers
+byte_offset       +insert_expand     +perl/dyn          +title
+channel           +ipv6              +persistent_undo   +toolbar
+cindent           +job               +popupwin          +user_commands
+clientserver      +jumplist          +postscript        +vartabs
+clipboard         +keymap            +printer           +vertsplit
+cmdline_compl     +lambda            +profile           +vim9script
+cmdline_hist      +langmap           -python            +viminfo
+cmdline_info      +libcall           +python3/dyn       +virtualedit
+comments          +linebreak         +quickfix          +visual
+conceal           +lispindent        +reltime           +visualextra
+cryptv            +listcmds          +rightleft         +vreplace
+cscope            +localmap          +ruby/dyn          +wildignore
+cursorbind        +lua/dyn           +scrollbind        +wildmenu
+cursorshape       +menu              +signs             +windows
+dialog_con_gui    +mksession         +smartindent       +writebackup
+diff              +modify_fname      -sodium            +X11
+digraphs          +mouse             +sound             -xfontset
+dnd               +mouseshape        +spell             +xim
-ebcdic            +mouse_dec         +startuptime       -xpm
+emacs_tags        +mouse_gpm         +statusline        +xsmp_interact
+eval              -mouse_jsbterm     -sun_workshop      +xterm_clipboard
+ex_extra          +mouse_netterm     +syntax            -xterm_save
+extra_search      +mouse_sgr         +tag_binary        
-farsi             -mouse_sysmouse    -tag_old_static    
   system vimrc file: "/etc/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  system gvimrc file: "/etc/gvimrc"
    user gvimrc file: "$HOME/.gvimrc"
2nd user gvimrc file: "~/.vim/gvimrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
    system menu file: "$VIMRUNTIME/menu.vim"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-4 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/lzo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cloudproviders -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/at-spi-2.0 -pthread -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -g -ffile-prefix-map=/build/vim/src=/usr/src/debug -flto=auto -D_REENTRANT -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 
Linking: gcc -L. -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -fstack-protector-strong -rdynamic -Wl,-export-dynamic -Wl,--no-as-needed -Wl,-E -Wl,-rpath,/usr/lib/perl5/5.36/core_perl/CORE -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -flto=auto -L/usr/local/lib -o vim -lgtk-3 -lgdk-3 -lz -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lSM -lICE -lXt -lX11 -lXdmcp -lSM -lICE -lm -ltinfo -lelf -lcanberra -lrt -lacl -lattr -lgpm -Wl,-E -Wl,-rpath,/usr/lib/perl5/5.36/core_perl/CORE -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -flto -fstack-protector-strong -L/usr/local/lib -L/usr/lib/perl5/5.36/core_perl/CORE -lperl -lpthread -ldl -lm -lcrypt -lutil -lc -L/usr/lib -ltclstub8.6 -ldl -lz -lpthread -lm
```

### Python version

3.10.5 (main, Jun  6 2022, 18:49:26) [GCC 12.1.0]

### Operating System

Linux redviper 5.18.7-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 25 Jun 2022 20:22:01 +0000 x86_64 GNU/Linux

### Declaration

- [X] I have read and understood [CONTRIBUTING.md](https://github.com/puremourning/vimspector/blob/master/CONTRIBUTING.md#diagnostics)
- [X] I have read and understood the [CODE_OF_CONDUCT.md](https://github.com/puremourning/vimspector/blob/master/CODE_OF_CONDUCT.md)

**Comment by puremourning** [comment]
wait just realised your config uses CodeLLDB. I'll try that

**Comment by puremourning** [comment]
still no repro. Are you using the latest CodeLLDB (e.g. VimspectorUpdate).

**Comment by Aster89** [comment]
Yes I have...

**Comment by puremourning** [comment]
I can repro, sorry I completely ignored one of your instructions "toggle the breakpoint in the breakpoints window" ; this is what causes it and it's fixed in #559 when the neovim tests deign to allow it to merge
